data/prev_events.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "title": "Introduction to VUI",
+    "date": "9th January",
+    "poster": "images/flashbox/vui_event_poster.png",
+    "startTime": "02:30 PM",
+    "endTime": "04:40 PM",
+    "description": "In this beginner friendly event learn what is VUI, how it functions and create your own VUI app using Alexa and Google Assistant without writing even a single line of code."
+  }
+]
```

Comment:
Wouldn't previous events include all previous events or are we considering only 2020. 

IMO it should extend to at least 2019

Reply:
I also think all the events should be added. But I had no information regarding the events.

I can add more now. Where can I find the previous events?

Reply:
Previous events data can be found in previous commits

Reply:
or file history